src/pages/Projects/components/RoleCard/RoleCard.tsx

```diff
@@ -89,7 +89,8 @@ export default function RoleCard({
             lineHeight="20px"
             letterSpacing="0.25px"
           >
-            {role.endConfidence && `${role.endConfidence * 100}%`}
+            {(role.endConfidence || role.endConfidence === 0) &&
```

Comment:
you could do `typeof role.endConfidence === 'number'`

Reply:
wouldn't null be considered a number?

Reply:
no, `typeof null` returns `"object"` not `"number"`